src/index.js

```diff
@@ -23,6 +25,8 @@ export { default as FieldLabel } from './components/field-label';
 
 export { default as TextField } from './components/fields/text-field';
 
+export { Icons };
+// TODO: this type of export is deprecated and should be removed in the next major release
 export * from './components/icons';
```

Comment:
This is kept for backwards compatibility and will be removed in the next major release